examples_tests/23.Autoexposure/CMakeLists.txt

```diff
@@ -5,8 +5,8 @@ if(NOT RES)
 endif()
 
 set(EXAMPLE_SOURCES
-	../../ext/LumaMeter/CLumaMeter.cpp
-	../../ext/ToneMapper/CToneMapper.cpp
+	../../src/irr/ext/LumaMeter/CLumaMeter.cpp
+	../../src/irr/ext/ToneMapper/CToneMapper.cpp
 )
 
```

Comment:
oh boy many more example cmakes needs to be updated then, make sure you run all examples.